extensions/vp9/src/main/java/com/google/android/exoplayer2/ext/vp9/LibvpxVideoRenderer.java

```diff
@@ -314,7 +314,7 @@ private void renderBuffer() {
       outputBuffer = null;
       consecutiveDroppedFrameCount = 0;
       decoderCounters.renderedOutputBufferCount++;
-      maybeNotifyRenderedFirstFrame();
+      maybeNotifyRenderedFirstFrame(0);
```

Comment:
Comments in MediaCodecVideoRenderer apply to this file too (but I haven't duplicated them all :)).